Doc/library/xmlrpc.client.rst

```diff
@@ -62,6 +62,10 @@ between conformable Python objects and XML on the wire.
    The obsolete *use_datetime* flag is similar to *use_builtin_types* but it
    applies only to date/time values.
```

Comment:
You must document header. Does it have to be a list? Can it be a set? Or any iterable?

Reply:
It is said in line 66 that it is a list.

Reply:
> It is said in line 66 that it is a list.

I would prefer to not document the parameter in the "versionchanged" markup, but in the "body" of the doc. versionchanged should basically say "new parameter".

You used. I would prefer to use versionchanged instead.

Just move the content of your versionadded into the body, and add something like "versionchanged:: 3.8 New *headers* parameter." (I let you format it properly ;-))